sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -0,0 +1,28 @@
+[id="understanding-sandboxed-containers"]
+= Understanding OpenShift sandboxed containers
+include::modules/common-attributes.adoc[]
+
+:context: understanding-sandboxed-container
+
+toc::[]
+
+[role="_abstract"]
+
+OpenShift sandboxed containers support for Red Hat OpenShift provides users seeking additional workload isolation a built-in support for running Kata containers as an additional optional runtime on OpenShift Container Platform. This is particularly useful for users who are wanting to perform the following tasks.
```

Comment:
This might actually be _Kata Containers_, all capitalized.

Reply:
It is! Thanks for catching this.